cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -137,7 +137,7 @@ std::vector<bitmask_type> make_null_mask_vector(ValidityIterator begin,
       cudf::bitmask_allocation_size_bytes(size) / sizeof(bitmask_type);
   std::vector<bitmask_type> null_mask(num_words, 0);
   for (auto i = 0; i < size; ++i) {
-    if (begin[i] == true) {
+    if (*(begin + i)) {
```

Comment:
I think the `[i]` version is clearer. Is there a reason to change this?

Reply:
Not all iterators support `operator[]`. This is definitely the more canonical way of handling iterators.

Reply:
IIRC I made this change specifically because of the fact that it was necessary.